content/en-US/api/2dcontextrestoreevent.md

```diff
@@ -0,0 +1,35 @@
+---
+recipe: api-interface
+title: '2dcontextrestored'
+mdn_url: /en-US/docs/Web/API/2dcontextrestored_event
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.2dcontextlost
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The `contextRestored` event is a javascript event that is triggered when the user agent
+detects that the context associated with 'CanvasRenderingContext2D' on the page is "restored"
+after being "lost".
```

Comment:
Here's [how an interface should be worded](https://developer.mozilla.org/en-US/docs/Web/API/MIDIPort).